### PR TITLE
Basic serpente port

### PR DESCRIPTION
**Pull request by EnJens**
This is a basic board with some examples for the Serpente board (https://serpente.solder.party/)
The basic blinky example is included as well as a PWM example with a cheap rainbow-y effect on the built-in RGB led.

### CODE REVIEW

**Comment by sajattack** [comment]
Can you add it to .travis.yml so I can see the test results?

**Comment by EnJens** [comment]
@sajattack My bad - done now :)

**Comment by sajattack** [comment]
Thanks

File: boards/serpente/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# @Arturo182 Serpente Board Support Crate
```
Comment (arturo182) [inline boards/serpente/README.md]: arturo182 is always spelled lowercase :D

**Comment by sajattack** [comment]
You can add your board to the table in the main README while you're tweaking the other ones.

**Comment by sajattack** [comment]
@arturo182 might want to update your blog CSS XD 
image

**Comment by arturo182** [comment]
Well, you know blogs, they die young ;)

**Comment by EnJens** [comment]
@sajattack should be all fixed now :)

**Comment by sajattack** [comment]
@EnJens thanks for all your hardwork and welcome a-board @arturo182

**Comment by sajattack** [comment]
I'll release this all on crates.io tomorrow.

**Pull request merged**